setup/conda/README.md

```diff
@@ -0,0 +1,21 @@
+# Environment and Tool Setup Using Conda
+
+The LiteX project [provides recipes](https://github.com/hdl/conda-eda) for installing many core open-source EDA tools using the Anaconda / Miniconda Python package management system.
```

Comment:
nit: technically, this is the `conda-eda` project under the `hdl` organization., the channel is still named `Litex-Hub` for legacy quota reasons.